ps/jenkins/package-testing-ps-build-5.7.groovy

```diff
@@ -0,0 +1,230 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+setup_rhel_package_tests = { ->
+    sh '''
+        sudo yum install -y epel-release
+        sudo yum -y update
+        sudo yum install -y ansible git wget
+    '''
+}
+
+setup_amazon_package_tests = { ->
+    sh '''
+        sudo amazon-linux-extras install epel
+        sudo yum -y update
+        sudo yum install -y ansible git wget
+    '''
+}
+
+setup_debian_package_tests = { ->
+    sh '''
+        sudo apt-get install -y dirmngr gnupg2
```

Comment:
it is better to execute `apt update` as the first command before installing anything as apt database might require update